ui/static/ui/js/listing.js

```diff
@@ -213,6 +229,72 @@ define(
         });
       });
 
+      var updateCheckDisplay = function(node) {
+        if ($(node).data("selected")) {
+          $(node).find("i").attr("class", "fa fa-check");
+        } else {
+          $(node).find("i").attr("class", "fa fa-arrow-right");
+        }
+      };
+      /**
+       * Update export count badge
+       * @param {number} direction Add this value to
+       * the existing count before rendering
+       */
+      var updateExportCount = function(direction) {
+        var oldCount = $("#export_button").data("export-count");
+        var newCount = oldCount + direction;
+
+        $("#export_button .badge").remove();
```

Comment:
I see. `var exports` is actually using React. It's receiving a list of React elements which will be provided to the div a few lines down from that. It looks like a template but it's just some syntactic sugar. Same for `errorBox` and `messageBox`